sparse_strips/vello_cpu/src/render.rs

```diff
@@ -84,33 +84,30 @@ impl Default for RenderSettings {
 impl RenderContext {
     /// Create a new render context with the given width and height in pixels.
     pub fn new(width: u16, height: u16) -> Self {
-        let settings = RenderSettings::default();
-        Self::new_inner(width, height, settings.num_threads, settings.level)
+        Self::new_inner(width, height, RenderSettings::default())
     }
 
     /// Create a new render context with specific settings.
     pub fn new_with(width: u16, height: u16, settings: &RenderSettings) -> Self {
-        Self::new_inner(width, height, settings.num_threads, settings.level)
+        Self::new_inner(width, height, *settings)
     }
 
-    fn new_inner(width: u16, height: u16, num_threads: u16, level: Level) -> Self {
+    fn new_inner(width: u16, height: u16, settings: RenderSettings) -> Self {
```

Comment:
I think we should just make this public:

```suggestion
    /// Create a new render context with specific settings.
    pub fn new_with(width: u16, height: u16, settings: RenderSettings) -> Self {
```